docs/zh/latest/getting-started.md

```diff
@@ -257,10 +257,10 @@ curl "http://127.0.0.1:9180/apisix/admin/routes/1" -H "X-API-KEY: edd1c9f034335f
 我们已经创建了路由与上游服务，现在可以通过以下命令访问上游服务：
 
 ```bash
-curl -i -X GET "http://127.0.0.1:9080/get?foo1=bar1&foo2=bar2" -H "Host: httpbin.org"
+curl -i -X GET "http://127.0.0.1:9080/anything/get?foo1=bar1&foo2=bar2" -H "Host: httpbin.org"
 ```
 
-该请求将被 APISIX 转发到 `http://httpbin.org:80/anything/foo?arg=10`。
+该请求将被 APISIX 转发到 `http://httpbin.org:80/anything/get?foo1=bar1&foo2=bar2`。
```

Comment:
These changes are incorrect.

Reply:
thanks for pointing out, I'll push changes for them :)